docs/workflows/outputs/debug.md

```diff
@@ -29,3 +33,11 @@ Optionally, you may also enable the Live Debug Stream by clicking the button in
 ![Debug Node Live Stream](/images/workflows/outputs/debug-live-view.png "Debug Node Live Stream")
 
 Note that, should multiple workflow runs occur per second, some of the runs may not be visualized on the stage. Also, if you have unsaved changes in your workflow, live streams will automatically be disabled until the changes are saved.
+
+## Cloud vs Edge Behavior
+
+If the workflow is a [cloud](/workflows/cloud-workflow) workflow, debug node output is generated for any runs of the workflow whenever you are looking at the workflow. For [edge](/workflows/edge-workflows/#debugging) workflows, however, by default no debug output is generated. When you want debug output for an edge workflow, you must first select an [edge compute](/devices/edge-compute/) device that the workflow is deployed and running on. At that point, Losant will tell the edge agent on that device to start debugging that workflow, and the edge agent will start reporting debugging information.
```

Comment:
should cloud workflow be linked instead of just cloud
same for edge workflow
capitalize edge compute
capitalize edge agent

Reply:
👍